layers/+web-services/twitch/packages.el

```diff
@@ -0,0 +1,52 @@
+;;; packages.el --- twitch layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Benedikt Broich <b.broich@posteo.de>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Code:
+
+(defconst twitch-packages
+  '(
+    ;; (erc-twitch :toggle (configuration-layer/package-usedp 'erc))
+    (twitch-api :location (recipe :fetcher github
+                                  :repo "benediktbroich/twitch-api"))
+    (helm-twitch :location (recipe :fetcher github
+                                   :repo "benediktbroich/helm-twitch")
+                 ;; :toggle (configuration-layer/package-usedp 'helm)
```

Comment:
One issue is this depends on `helm` so you indeed need some "toggle".
Here you can use `:requires`.